CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Changelog for Critical Maps iOS
 
 ### Fixed
 
+- SocialSegment sliding under NavigationBar bug in iOS 10
```

Comment:
This should be added to Unreleased

Reply:
Ups. Thought I did

Reply:
Think it is:
https://github.com/mltbnz/criticalmaps-ios/blob/socialSelectionFix/CHANGELOG.md